main.go

```diff
@@ -2,16 +2,30 @@ package main
 
 import (
 	"./logging"
+	"fmt"
 	"log"
 	"os"
 	"strings"
+	"sync"
+
+	"github.com/hashicorp/go-reap"
 )
 
 func main() {
 	log.SetOutput(os.Stdout)
 	if strings.EqualFold(os.Getenv("DEBUG"), "true") {
 		go logging.StartLogging()
 	}
+	// Get feedback on reaped children and errors.
+	if reap.IsSupported() {
+		pids := make(reap.PidCh, 2)
+		errors := make(reap.ErrorCh, 2)
+		done := make(chan struct{})
+		var reapLock sync.RWMutex
+		go reap.ReapChildren(pids, errors, done, &reapLock)
```

Comment:
sounds like a good approach to test. Thanks so much for your thoughts. I somehow can't get the IntegrationTests running. The unit test run without a problem. This is the output:

```
=== RUN   TestGeneralIntegrationSwarmTestSuite
2017/06/25 13:34:31 exit status 1
FAIL	_/usr/src/myapp/integration_tests	10.594s
```

Any idea why this fails or how to get a more detailed output?